workflows/src/main/java/de/uniks/backend/services/FulibWorkflowsService.java

```diff
@@ -0,0 +1,165 @@
+package de.uniks.backend.services;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import de.uniks.backend.model.GenerateResult;
+import org.apache.tomcat.util.http.fileupload.ByteArrayOutputStream;
+import org.fulib.workflows.generators.BoardGenerator;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipOutputStream;
+
+public class FulibWorkflowsService {
+    Logger logger = LoggerFactory.getLogger(FulibWorkflowsService.class);
+
+    public String generate(String yamlData) {
+        GenerateResult generateResult = generateFromYaml(yamlData, true);
+
+        String answer = "";
+
+        ObjectMapper objectMapper = new ObjectMapper();
+        try {
+            answer = objectMapper.writeValueAsString(generateResult);
+        } catch (JsonProcessingException e) {
+            e.printStackTrace();
+        }
+
+        return answer;
```

Comment:
Can you not return generateResult here and let Spring handle the JSON conversion?

Reply:
Returning the generateResult works like a charm and the redundant JSON conversion got removed.